NC-2740: Update Turnstile Documentation to reflect some callback changes (#8517)
cloudflare/cloudflare-docs: content/turnstile/_index.md

Customers are able to choose the widget type via Turnstile. This helps avoid [CAPTCHAs](https://www.cloudflare.com/learning/bots/how-captchas-work/), which also reduces the lifetimes of human time spent solving CAPTCHAs across the Internet.

> Turnstile can be embedded into various shapes or forms in an application from a visual widget into a completely invisibile widget running in the background.

Reviewer: ```suggestion
Turnstile can be embedded in various forms in an application, from a visual widget to a completely invisible widget running in the background.
```
